packages/core/src/button/button.stories.ts

```diff
@@ -229,10 +229,10 @@ export function sizes() {
 export function block() {
   return html`
     <div cds-layout="vertical gap:sm align:horizontal-stretch">
-      <cds-button block>Default ('md')</cds-button>
-      <cds-button block action="outline">Default ('md')</cds-button>
-      <cds-button block size="sm">Compact ('sm')</cds-button>
-      <cds-button block action="outline" size="sm">Compact ('sm')</cds-button>
+      <cds-button>Default ('md')</cds-button>
```

Comment:
Are we no longer supporting `block` buttons?

Reply:
We could leave the attr in the css for now as a deprecation. The demos work without block because the buttons work correctly with flexbox and the layout utils

Reply:
Reverted the block attr removal then updated the demos to use block and not the layout stretch util